build.gradle

```diff
@@ -34,11 +34,12 @@ description = 'New code for Egeria'
 
```

Comment:
the ext.name and description above should refer to this connector

Reply:
it's worth checking the generated manifest for the maven artifact. The group is specified in the top level gradle. Names can default to directory name, but as you mention settings etc can override. Worth a second iteration once this is merged